allahcalc.mk

```diff
@@ -0,0 +1,26 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS += allahcalc
+ALLAHCALC_VERSION := 1.1
+DEB_ALLAHCALC_V ?= $(ALLAHCALC_VERSION)
+
+allahcalc-setup: setup
+	wget -q -nc -P (BUILD_SOURCE) https://github.com/clyde37/allah-calculator/releases/download/$(ALLAHCALC_VERSION)/allah_calculator_generator.cpp
+
+ifneq ($(wildcard $(BUILD_WORK)/allahcalc/.build_complete),)
+allahcalc:
+	@echo "Using previously built allahcalc."
+else
+allahcalc: allahcalc-setup
+    gcc++ allah_calculator_generator.cpp -o allahcalc/allah_calculator_generator
+    ./allah_calculator_generator
+    gcc allah_calculator_final.mp4_custom.c -o allahcalc/allahcalc
+endif
+
+allahcalc-package: allahcalc-stage
+    $(call PACK,allahcalc,DEB_ALLAHCALC_V)
+    rm -rf allahcalc
```

Comment:
You should know where files get put, so fix this to use the proper directories.

Reply:
not sure what that means

Reply:
If you know what ``$(BUILD_SOURCE)`` is for, you should know that, as of right now, this entire thing won't compile at all. You need to move your files to specific places before compiling them.